tv_apps/smart-system/js/homescreen_launcher.js

```diff
@@ -24,6 +25,16 @@
   var HomescreenLauncher = function() {
   };
 
+  HomescreenLauncher.EVENTS = [
```

Comment:
Yes, you're right! I misunderstood the `BaseModule.mixin()`. I'll look into it soon.